wpcc.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'dough-ruby'
+  s.add_dependency 'dough-ruby', '~> 5.25'
```

Comment:
@margOnline do you mean how should the information to update `dough-ruby` be communicated? If that's the question then I say it should be updated whenever someone touches a tool that uses/depends on dough. 

It'll be difficult to imagine having to update every tool when there's a new version of the gem. It's impractical.